src/components/FavoriteView.jsx

```diff
@@ -1,8 +1,25 @@
 import React, { useState, useEffect } from 'react';
+import RecipeTile from './RecipeTile.jsx'
+
 
 const FavoriteView = () => {
 
   return (
+    <div id="favoriteMain">
+      <div id="favoriteTitle">Favorite Recipes</div>
+      <div id="favoriteList">
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
```

Comment:
I assume these are just placeholders until we actually load user information?

Reply:
yes it's just placeholders before we load the actual data.